File: completion-plugin/testData/java/SwitchStatement.java
public class SwitchStatement {
    public static void main(String[] args) {
        switch (args.length) {
            case 0:
                test("No arguments found");
                break;
            case 1:
                test(args[0]);
            default:
                test("Unexpected arguments");
        }
    }


    private void test(Object a) {
    }
}

File: completion-plugin/testData/csharp/SwitchStatement.cs
using System;

namespace test
{
    public class SwitchStatement
    {
        public static void Main(string[] args)
        {
            switch (args.Length)
            {
                case 0:
                    Console.WriteLine("No arguments found");
                    break;
                case 1:
                    Console.WriteLine(args[0]);
                    break;
                default:
                    Console.WriteLine("Unexpected arguments");
                    break;
            }
        }
    }
}
